README.md

```
@@ -1,6 +1,228 @@
-# dnsmenace
-DNS Menace will allow you to send a dns query to any name server in the world based on the two letter country code.
+# DNSMenace
 
-ex.
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)